cmd/probe.go

```diff
@@ -35,4 +35,5 @@ func init() {
 	probeCmd.Flags().StringVarP(&probeInstallOptions.Namespace, "namespace", "n", "kube-system", "Namespace for resources")
 	probeCmd.Flags().BoolVar(&probeInstallOptions.Full, "full", false, `If KubeArmor is not running, it deploys a daemonset to have access to more
 information on KubeArmor support in the environment and deletes daemonset after probing`)
+	probeCmd.Flags().StringVarP(&probeInstallOptions.Output, "format", "f", "", " Format: json or text ")
```

Comment:
I suggest you have default option as text. If required we can provide json

Reply:
since there are no other export options can we keep it as a boolean flag `--json` @seswarrajan what do you think?

Reply:
Good suggestion @vishnusomank 
What if in future, we plan to extend the output format as anything else. 

In that case we can just add one new option and add code for the same.
Someone else's suggestion is also welcome on the same.

Reply:
> I suggest you have default option as text. If required we can provide json

Yes, by default it shows output as text only, i.e if there are no flags provided